lib/auth/client.dart

```diff
@@ -10,6 +10,7 @@ class VerboseClient extends http.BaseClient {
 
   @override
   Future<http.StreamedResponse> send(http.BaseRequest request) async {
+    print("VerboseClient send");
```

Comment:
Is this line really necessary? I prefer to avoid verbose output whenever possible.

Reply:
No need, I'm sorry for the initial contribution.